protoc-gen-java-snowpea/src/main/resources/classTemplate.vm

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright ${year} Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+ 
+// Code generated by protoc-gen-java-snowpea. DO NOT EDIT.
+
+package com.${packageName};
+
+import static org.junit.Assert.assertTrue;
+
+import com.google.protobuf.util.JsonFormat;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+#if($skip || $skipStrict)
```

Comment:
In Go with feedback from Marc I shifted to a three-mode test scheme: loose, strict, and compatibility. Pub/Sub and Audit Logs skip on strict and have a compatibility test where a special data preparation function is called. All other types skip the compatibility test. This was to help emphasize the special casing going on.

I have kept the cleanup functions internal for now, but I expect they may end up being opened as part of samples work.

Reply:
Captured here: https://github.com/googleapis/google-cloudevents-java/issues/129